GOVERNANCE.md

```diff
@@ -0,0 +1,177 @@
+
+Each level unlocks new privileges and responsibilities on Discord and GitHub. Below is a summary of each contributor level:
+
+### Level 1 - Contributor
+
+Have you done something (big or small) to contribute to the health, success, or growth of the GUC or its plugins? Congratulations, you're officially recognized as a contributor to the project!
+
+#### Examples of recognized contributions
+
+- **GitHub:** Submitting a merged pull request
+- **GitHub:** Filing a detailed bug report or RFC
+- **GitHub:** Updating documentation!
+- Helping people on GitHub, Discord, etc.
+- Answering questions on Stack Overflow, Twitter, etc.
+- Blogging, Vlogging, Podcasting, and Livestreaming about GUC projects
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privileges
+
+At this time we have no specifics for this role. as the community grows we anticipate privleges to come. If you have ideas here please let us know!
+
+#### Responsibilities
+
+This role does not require any extra responsibilities or time commitment. We hope you stick around and keep participating!
+
+If you're interested in reaching the next level and becoming a Maintainer, you can begin to explore some of those responsibilities in the next section.
+
+#### Nomination Process
+
+n/a
+
+### Level 2 (L2) - Maintainer
+
+The **Maintainer** role is available to any contributor who wants to join the team and take part in the long-term maintenance of a plugin(s) in the GUC.
+
+The Maintainer role is critical to the long-term health of GUC plugins. Maintainers act as the first line of defense when it comes to new issues, pull requests and Discord activity. Maintainers are most likely the first people that a user will interact with on Discord or GitHub.
+
+**A Maintainer is not required to write code!** Some Maintainers spend most of their time inside of Discord, maintaining a healthy community there. Maintainers can also be thought of as **Moderators** on Discord and carry special privileges for moderation.
+
+There is no strict minimum number of contributions needed to reach this level, as long as you can show **sustained** involvement over some amount of time (at least a couple of weeks).
+
+#### Recognized Contributions
+
+- **GitHub:** Contributing a new or existing Gatsby plugin
+- **GitHub:** Submitting non-trivial pull requests and RFCs
+- **GitHub:** Reviewing non-trivial pull requests and RFCs
+- **Discord:** Supporting users in Discord
+- **Discord:** Active participation in calls and other events
+- **GitHub + Discord:** Triaging and confirming user issues
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privileges
+
+- All privileges of the [Contributor role](#level-1---contributor), plus...
+- `@maintainer` role on [Discord](https://discord.gg/gwnqfzetjv)
+- New name color on Discord: **blue**.
+- Invitation to the private #maintainers channel on Discord.
```

Comment:
```suggestion
- Invitation to the private `#maintainers` channel on Discord.
```

@pragmaticpat we want us (Gatsby) commit ourselves to also be in that channel and help out as needed. As in that sentence would also say: "[...] where also the Gatsby team can be reached"

Reply:
@LekoArts - I don't see which sentence you're specifically referencing. That said, I agree that there can be a mention of a Gatsby team member that at least checks in on some frequency. We should probably consider some sense of "office hours" for supporting any questions that arise. 

How's that sound to you and @moonmeister ?

Reply:
Doesn't sound like that needs to be in the governance docs. If we want to mention this as a resource in the contributing docs that seems fine. Unless we want to give maintainers explicit access to the Gatsby team.

Reply:
> If we want to mention this as a resource in the contributing docs that seems fine.

👍